jenkins/runlocal

```diff
@@ -1,52 +1,35 @@
 #!/bin/bash -e
-
+#
 # Build the Pbench Server RPM and container, and run functional tests locally.
-# Requires a Fedora, CentOS, or RHEL environment to run.
-
+#
 # NOTE WELL: By default, when the functional tests are run, the infrastructure
 # pod and Pbench Server container are left running by default.  Add the switch,
 # `--cleanup` to direct `jenkins/run-server-func-tests` to cleanup when
 # finished (success or failure).
 
-# Build the pbench-server RPM locally, then build the containers locally, and
-# then run the functional tests against the locally built CI container image.
 export PB_SERVER_IMAGE_NAME=pbench-server
 
 # We use the current user name as the tag to avoid any conflict with what the CI
 # environment does.
+export PB_CONTAINER_REG=images.paas.redhat.com/pbench
 export PB_SERVER_IMAGE_TAG=${USER}
 
 # We use the image pull policy of `never` here to ensure our locally built image
 # is used by the pod.
 export PB_SERVER_IMAGE_PULL_POLICY=never
 
 # Create an RPM from the current source tree and double check it exists.
-make -C server/rpm clean rpm
-export RPM_PATH=${HOME}/rpmbuild/RPMS/noarch/pbench-server-*.rpm
+# Set the workspace to the home directory so that the RPM built inside the
+# container will be available after it exits.
+WORKSPACE_TMP=${WORKSPACE_TMP:-${HOME}}
```

Comment:
Shouldn't this be an `export`?

Reply:
Yes.  I wonder how it's been working without that....  🤯

Reply:
> I wonder how it's been working without that...

I was defining it on my `runlocal` command line.  🦐